R/RLum.Analysis-class.R

```diff
@@ -867,15 +867,19 @@ setMethod(
 #'
 #' Sorting of `RLum.Data` objects contained in this `RLum.Analysis` object.
 #' At least one of `slot` and `info_element` must be provided. If both are
-#' given, ordering by `slot` always supersedes ordering by `info_element`.
+#' given, ordering by `slot` always takes priority over `info_element`.
 #'
 #' @param slot [character] (*optional*): slot name to use in sorting.
 #'
-#' @param info_element [character] (*optional*): name of the `info` field
-#' to use in sorting.
+#' @param info_element [character] (*optional*): names of the `info` field
+#' to use in sorting. The order of the names sets the sorting priority.
+#' Regardless of available info elements, the following
+#' elements always exist because they are calculated from the record `XY_LENGTH`, `X_MIN`,
+#' `X_MAX`, `Y_MIN`, `Y_MAX`
 #'
-#' @param decreasing [logical] (*with default*): whether the sort order should
-#' be decreasing (`FALSE` by default).
+#' @param descending [logical] (*with default*): whether the sort order should
```

Comment:
I had called it `decreasing` to match what R uses in `sort` and `order`.

Reply:
Ah, OK, if to admit that I was had the impression that "ascending" and "descending" is used more often. I will change that back.